Core/Utilities/Subprocess.py

```diff
@@ -368,6 +368,9 @@ def __readFromFile( self, fd, baseLength, doAll = True ):
         if nB == "":
           break
         dataString += nB
+        if len( dataString ) > 10000000:
```

Comment:
Just for the posterity, rather than hardcoding it here, could you define a global varaible with a meaningful name (and eventually comment :-) ) so that in the future we remember why this limitation is here ?